pyhf/pdf.py

```diff
@@ -131,6 +132,9 @@ def __init__(self, spec, **config_kwargs):
         # build up our representation of the specification
         self.config = _ModelConfig.from_spec(self.spec,**config_kwargs)
 
+        self.prepped_constraints_gaussian = gaussian_constraint_combined(self)
```

Comment:
i don't like passing in the `pdf` here, but maybe it's fine. It just makes the call look awkward of `self.... = constraint_combined(self)`.

Reply:
I think we only depend on pdf config.. so we could change that to `gaussian_constraint_combined(self.config)`